nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -140,11 +140,15 @@ def get_start_nodes_for_activation_path_tracing(nncf_graph: PTNNCFGraph) -> List
 
     @staticmethod
     def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> PTTargetPoint:
+        input_port_id: Optional[int] = port_id
         if NNCFGraphNodeType.INPUT_NODE in target_node_name or target_type == TargetType.POST_LAYER_OPERATION:
-            port_id = None
-        if target_type in PTMinMaxAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP:
+            input_port_id = None
+        if (
+            not is_experimental_torch_tracing_enabled()
+            and target_type in PTMinMaxAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP
+        ):
             target_type = PTMinMaxAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP[target_type]
-        return PTTargetPoint(target_type, target_node_name, input_port_id=port_id)
+        return PTTargetPoint(target_type, target_node_name, input_port_id=input_port_id)
```

Comment:
What do you think about the same interpretation of the following target types:
- `TargetType.PRE_LAYER_OPERATION and TargetType.OPERATOR_PRE_HOOK`
- `TargetType.POST_LAYER_OPERATION: TargetType.OPERATOR_POST_HOOK`

I believe this is simplify integration of the torch2function tracing.

Reply:
Think that it can not simplify integration